### DISCUSSION TITLE
Programmier-Lession 04.12.2023

### DISCUSSION THREAD

**Discussion by DGrillCoder** [grilldb97/legendary-space-zebra#1] [Show and tell]
Erste Stunde in der Programmierung :D an der DH
Toller Einstieg

for Schleife in verschiedene Programmiersprachen getestet:

z.b. Java:
public class Main {

public static void main (String[] args){
for (int i = 0; i <= 10; i = i + 2) {
  System.out.println(i);
  }
 }
}

**Comment by JonNoe** [comment]
Toller Code 👍

**Comment by SirPatschiii** [comment]
Looks wonderful Dustin :D
